media-gfx/rawtherapee/rawtherapee-5.4.ebuild

```diff
@@ -0,0 +1,66 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils toolchain-funcs flag-o-matic
+
+DESCRIPTION="A powerful cross-platform raw image processing program"
+HOMEPAGE="http://www.rawtherapee.com/"
+
+MY_P=${P/_rc/-rc}
+SRC_URI="http://rawtherapee.com/shared/source/${MY_P}.tar.xz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+IUSE="openmp"
+
+RDEPEND="x11-libs/gtk+:3
+	dev-libs/expat
+	dev-libs/libsigc++:2
+	media-libs/libcanberra[gtk3]
+	media-libs/tiff:0
+	media-libs/libpng:0
+	media-libs/libiptcdata
+	media-libs/lcms:2
+	media-libs/lensfun
+	sci-libs/fftw:3.0
+	sys-libs/zlib
+	virtual/jpeg:0"
+DEPEND="${RDEPEND}
+	app-arch/xz-utils
+	virtual/pkgconfig
+	dev-cpp/gtkmm:3.0"
+
+S="${WORKDIR}/${MY_P}"
+
+pkg_pretend() {
+	if use openmp ; then
+		tc-has-openmp || die "Please switch to an openmp compatible compiler"
+	fi
+	# https://bugs.gentoo.org/show_bug.cgi?id=606896#c2
+	# https://github.com/vivo75/vivovl/issues/2
+	if [[ $(get-flag -O3) != "-O3" ]] ; then
```

Comment:
Yes, but you -O3 is the greater optimization supported by compilers since a long time.
And it's only a warning (requested by upstream) I hope this is the best compromise

Reply:
Actually, clang supports `-O4` for a long time. That said, I'm against giving suggestions like this. Gentoo does not encourage increasing `-O` level for individual packages, and I don't really want every third package to tell users to use `-O3` because things might get faster.

Reply:
ok then, dropped the -O3 stuff